aztec/src/main/kotlin/org/wordpress/aztec/formatting/BlockFormatter.kt

```diff
@@ -166,6 +166,16 @@ class BlockFormatter(editor: AztecText, val listStyle: ListStyle, val quoteStyle
         removeBlockStyle(textFormat, selectionStart, selectionEnd, makeBlock(textFormat, 0).map { it -> it.javaClass })
     }
 
+    fun <T : IAztecBlockSpan> removeEntireBlock(type: Class<T>) {
+        val bounds = getBoundsOfText(editableText, selectionStart, selectionEnd)
```

Comment:
Would you mind adding some comments for future reference?
Also, I'm not not sure I completely understood it. Can't we just get `BlockQuote` spana that fall in `selectionStart, selectionEnd` and remove them?

Reply:
Nice catch, this is redundant.

Reply:
Is the call to `val bounds = getBoundsOfText(editableText, selectionStart, selectionEnd)` required? 
Can't we use `selectionStart,selectionEnd` when getting the spans like the snippet below, and avoid that call?

```
    fun <T : IAztecBlockSpan> removeEntireBlock(type: Class<T>) {
        editableText.getSpans(selectionStart,selectionEnd, type).forEach {
            editableText.removeSpan(it)
        }
    }
```


I also see testRemoveQuoteFormatting failing. Maybe we need to refresh it?